Library/ViewModels/SettingsViewModel.swift

```diff
@@ -675,22 +690,23 @@ private enum Notification {
     case .creatorTips:                              return "Creator tips"
     case .follower, .mobileFollower:                return "New followers"
     case .friendActivity, .mobileFriendActivity:    return "Friend backs a project"
-    case .messages, .mobileMessages:                 return "New messages"
+    case .messages, .mobileMessages:                return "New messages"
     case .postLikes, .mobilePostLikes:              return "New likes"
     case .updates, .mobileUpdates:                  return "Project updates"
     }
   }
 }
 
 private enum Privacy {
-
   case following
   case recommendations
+  case showPublicProfile
 
   fileprivate var trackingString: String {
     switch self {
     case .following: return Strings.Following()
     case .recommendations: return Strings.Recommendations()
+    default: return ""
```

Comment:
should we add one for `showPublicProfile`? (I'm not sure)

Reply:
I checked with Nneka and we won't be tracking privacy switch changes, so no need to add one.